lib/api/core/plugins/pluginContext.js

```diff
@@ -21,44 +21,38 @@ function PluginContext (kuzzle) {
     this.errors[_.upperFirst(name)] = constructor;
   });
 
-  /*
-   lazy-loading accessors, accessible only to plugins loaded by
-   a Kuzzle Server instance
-    */
-  if (kuzzle.isServer) {
-    Object.defineProperty(this.accessors, 'passport', {
-      enumerable: true,
-      get: function () {
-        return {
-          use: kuzzle.passport.use
-        };
-      }
-    });
+  Object.defineProperty(this.accessors, 'passport', {
```

Comment:
We still need to only inject accessors if the context is built by a Kuzzle instance. Because there is still the matter of threaded plugins, and these injections will make their initialization crash because they can't provide a complete Kuzzle object to this constructor (see lib/core/plugins/workerReady.js)